src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -56,6 +56,54 @@ public void CantWriteToNonWritableStream(bool formatted, bool skipValidation)
             Assert.Throws<ArgumentException>(() => new Utf8JsonWriter(stream, options));
         }
 
+        [Fact]
+        public static void WritingStringsWithCustomEscaping()
+        {
+            var output = new ArrayBufferWriter<byte>();
+            var writerOptions = new JsonWriterOptions { Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping };
+
+            using (var writer = new Utf8JsonWriter(output))
+            {
+                writer.WriteStringValue("\u6D4B\u8A6611");
```

Comment:
`U+6D4B` and `U+8A66` might be allowed through unescaped in a future version, which could break this unit test. If you're looking for something stable that's highly unlikely to ever be allowed through unescaped, consider something from the range `U+E000..U+F8FF` (inclusive). That block is permanently reserved for private use and I highly doubt even the "relaxed" escaper will ever allow those to pass through unescaped.

If the test doesn't rely on this being output escaped or unescaped, you're good to go. :)

Reply:
> might be allowed through unescaped in a future version

We have a few writer tests that ensure the default behavior escapes certain characters. If we change that behavior, we would have to/should change those tests as well, so I would prefer the tests break at that time.